docs/uml-class-diagrams/database/test/DatabaseCommManagerImplementationTests/DatabaseCommManagerImplementationTests.puml

```diff
@@ -2,6 +2,12 @@
 
 package edu.oakland.test.database {
     class DatabaseCommManagerImplementationTests {
+        + void rfidInIsRfidOut()
+        + void offsetMatchDataPoint()
+        + void offsetMatchTrackData()
+        + void trackDataInIsTrackDataOut()
+        + void databasePersistentStorageIsNotNull() 
+        //Should there be a mode request?
 
```

Comment:
Regarding the request mode, I don't think that Middleware_01 makes any decisions off of what mode it currently is, but if it is required, we can borrow the implementation and testing from the `DisplayGisManager` and jam it in here if need be. We'll deal with that when we get there.
```suggestion
```